source/server/configuration_impl.cc

```diff
@@ -17,7 +17,9 @@
 #include "common/ratelimit/ratelimit_impl.h"
 #include "common/ssl/context_config_impl.h"
 #include "common/tracing/http_tracer_impl.h"
+#ifndef DISABLE_LIGHTSTEP_TRACING
 #include "common/tracing/lightstep_tracer_impl.h"
++#endif // DISABLE_LIGHTSTEP_TRACING
```

Comment:
:/ It would seem that I patched over from g3 but didn't actually build. I am cleaning this up and a couple of other things.